src/main/java/domain/LottoRank.java

```diff
@@ -0,0 +1,42 @@
+package domain;
+
+import java.util.Arrays;
+import java.util.List;
+
+public enum LottoRank {
+    FIRST(2_000_000_000, 6),
+    SECOND(30_000_000, 5),
+    THIRD(1_500_000, 5),
+    FOURTH(50_000, 4),
+    FIFTH(5_000, 3),
+    FAILED(0, 0);
+
+    private final int prize;
+    private final int matchCount;
+
+    LottoRank(int prize, int matchCount) {
+        this.prize = prize;
+        this.matchCount = matchCount;
+    }
+
+    public static LottoRank findLottoRank(int matchCount, boolean isBonus) {
```

Comment:
```suggestion
    public static LottoRank find(int matchCount, boolean isBonus) {
```

또는 

```suggestion
    public static LottoRank of(int matchCount, boolean isBonus) {
```

와 같은 네이밍과 어떠한 차이가 있을까요?